Tasks/NuGetCommand/task.json

```diff
@@ -81,7 +81,7 @@
             "name": "selectOrConfig",
             "type": "radio",
             "label": "Feeds to use",
-            "defaultValue": "config",
+            "defaultValue": "select",
```

Comment:
The internal keys are not localized since they are not user visible, and it seems the values are not localized.